src/index.html

```diff
@@ -7,12 +8,40 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
Browsers do not natively support SCSS files. Ensure that `index.scss` is compiled to CSS before being linked in the HTML. You might want to use a build tool like Webpack or a task runner like Gulp to automate this process.